azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/model/WebContainer.java

```diff
@@ -82,4 +83,16 @@ public boolean equals(Object target) {
     public int hashCode() {
         return Objects.hash(value);
     }
+
+    @Override
+    public String toString() {
+        if (this.equals(JAVA_OFF)) {
```

Comment:
do we need to handle `JAVA_OFF` here? If so, do we also need to handle `JAVA_OFF` in `JavaVersion`

Reply:
May modify value in constructor and return value directly in `toString`